src/lib/Scenes/City/City.tsx

```diff
@@ -62,14 +63,6 @@ export class CityView extends Component<Props, State> {
   scrollViewVerticalStart = 0
   scrollView: ScrollView = null
 
-  filters: Tab[] = [
-    { id: "all", text: "All" },
-    { id: "saved", text: "Saved" },
-    { id: "fairs", text: "Fairs" },
-    { id: "galleries", text: "Galleries" },
-    { id: "museums", text: "Museums" },
-  ]
-
```

Comment:
these all live in a separate place, which means we can set the default (and that it can be shared across the map and the city page